index.js

```diff
@@ -43,15 +44,15 @@ __SUPER_SECRET_CONTAINER_DEBUG_HOOK__(container => {
 		const group = UNSTATED.isCollapsed ? console.groupCollapsed : console.group;
 		group(name);
 
-		if (diff.added) {
+		if (!isEmpty(diff.added)) {
```

Comment:
You don't really need Lodash for this. Can just use `if (Object.keys(diff.added).length > 0) {`